wiki/History_of_osu!/April_Fools/en.md

```diff
@@ -0,0 +1,294 @@
+
+---
+
+## 2012
+
+### Flashlight/Hidden mod on website
+
+On 1 April 2012, the entire osu! website had a 50% chance of either the [Flashlight (FL) mod](/wiki/Game_modifier/Flashlight) or [Hidden (HD) mod](/wiki/Game_modifier/Hidden) being "enabled" on the website per page load (3/10 for HD; 1/5 for FL). A modern recreation of what the prank looked like for users at the time is shown below [[14,15,16,17,18,19]][r].
+
+![](img/2012-flashlight.gif "What the website would have looked like with the Flashlight mod enabled")
+
+![](img/2012-hidden.gif "What the website would have looked like with the Hidden mod enabled")
+
+### Bad Apple ranking chart
+
+The "Bad Apple Ranking Chart" was a joke ranking chart based on a variety of beatmaps that featured the song "Bad Apple!!" as part of osu!'s 2012 April Fools' joke. Announced via [a newspost](https://osu.ppy.sh/community/forums/posts/1431905) on 1 April 2012, the ranking chart actually functioned as a real working chart at the time and displayed the top 40 players who obtained the highest [ranked score](/wiki/Score#ranked-score) in any one of the handpicked selections of "Bad Apple!!" beatmaps [[20,21,22]][r]. An excerpt from the aforementioned newspost is inscribed below:
+
+> We decided to dedicate this chart to the greatest song and video ever created, Bad Apple!!. You can find the awesome chart here
+>
+> Now, as this is a chart of epic proportions, we needed to make our prize a little better this time! The winners will receive a Lily White poster and a hand drawn picture of Reimu. (A printer is required to receive these prizes)
+>
+> Look forward to our next charts, the Renai Circulation and Irony charts!
+
+— Cyclone, "Bad Apple Ranking Chart!" [[21,22]][r]
+
+The chart was opened on 1 April 2012 and closed on 2 April 2012. At the end of the ranking period, [Mesita](https://osu.ppy.sh/users/201459) was the number one player with a ranked score of 145,623,328 [[23]][r].
+
+The beatmaps included in the ranking chart are listed below:
+
+- [nomico - Bad Apple!! (James)](https://osu.ppy.sh/beatmapsets/6252)
+- [REDALiCE - Bad Apple!! (Rena-chan)](https://osu.ppy.sh/beatmapsets/10353)
+- [Masayoshi Minoshima ft. nomico - Bad Apple!! (Ephemeral)](https://osu.ppy.sh/beatmapsets/10435)
+- [Masayoshi Minoshima ft. nomico - Bad Apple!! (ignorethis)](https://osu.ppy.sh/beatmapsets/13177)
+- [Masayoshi Minoshima feat. StrawbellyCake - Bad Apple!! (German Version) (Larto)](https://osu.ppy.sh/beatmapsets/13664)
+- [Masayoshi Minoshima feat. Larto & nomico - Awesome Apple!! (Larto)](https://osu.ppy.sh/beatmapsets/14475)
+- [Masayoshi Minoshima feat. nomico - Bad Apple!! (ouranhshc)](https://osu.ppy.sh/beatmapsets/18260)
+- [Spiritsoulxx - Bad Apple!! (Tony)](https://osu.ppy.sh/beatmapsets/23760)
+- [Kommisar - Bad Apple!! (Chiptune Ver.) (Sushi)](https://osu.ppy.sh/beatmapsets/28222)
+- [Kalafina - Bad MagiApple (Makar8000)](https://osu.ppy.sh/beatmapsets/32003)
```

Comment:
There is this one too https://osu.ppy.sh/beatmapsets/299000#osu/670892

Reply:
afaict that map wasn't listed in the official rankings

Reply:
What do you mean by "official ranking"?

Reply:
Those mapsets were listed in the [Beatmap Charts](https://osu.ppy.sh/rankings/osu/charts?spotlight=50) of that time. Unfortunately, Neil's Bad Apple couldn't make it there.

Reply:
Ah okay my bad I thought it was like all the maps that were ranked on that period